modules/runtime-common/src/main/scala/com/snowplowanalytics/snowplow/runtime/Telemetry.scala

```diff
@@ -60,47 +61,40 @@ object Telemetry {
       val stream = for {
         uuid <- Stream.eval(Async[F].delay(UUID.randomUUID))
         sdj = makeHeartbeatEvent(config, appInfo, uuid)
-        tracker <- Stream.resource(initTracker(config, appInfo.name, httpClient))
+        req = makeHeartbeatRequest[F](config, sdj, appInfo.name)
         _ <- Stream.unit ++ Stream.fixedDelay[F](config.interval)
-        _ <- Stream.eval(tracker.trackSelfDescribingEvent(unstructEvent = sdj))
-        _ <- Stream.eval(tracker.flushEmitters())
+        _ <- Stream.eval(executeHttpRequest(httpClient, req))
       } yield ()
 
       stream.drain
     }
 
-  private def initTracker[F[_]: Async](
+  private def makeHeartbeatRequest[F[_]](
     config: Config,
-    appName: String,
-    client: Http4sClient[F]
-  ): Resource[F, Tracker[F]] =
-    for {
-      implicit0(random: Random[F]) <- Resource.eval(Random.scalaUtilRandom)
-      emitter <- Http4sEmitter.build(
-                   EndpointParams(config.collectorUri, port = Some(config.collectorPort), https = config.secure),
-                   client,
-                   retryPolicy = RetryPolicy.MaxAttempts(10),
-                   callback    = Some(emitterCallback[F] _)
-                 )
-    } yield new Tracker(NonEmptyList.of(emitter), "telemetry", appName)
+    sdj: SelfDescribingData[Json],
+    appName: String
+  ): Request[F] = {
+    val uri = (config.collectorUri / "com.snowplowanalytics.iglu" / "v1")
+      .copy(query = Query("aid" -> Some(appName), "p" -> Some("srv")))
+    Request[F](Method.POST, uri).withEntity(sdj.normalize)
+  }
 
-  private def emitterCallback[F[_]: Sync](
-    params: EndpointParams,
-    req: Request,
-    res: TrackerResult
+  private def executeHttpRequest[F[_]: Async](
+    httpClient: Http4sClient[F],
+    request: Request[F]
   ): F[Unit] =
-    res match {
-      case TrackerResult.Success(_) =>
-        Logger[F].debug(s"Telemetry heartbeat successfully sent to ${params.getGetUri}")
-      case TrackerResult.Failure(code) =>
-        Logger[F].warn(s"Sending telemetry heartbeat got unexpected HTTP code $code from ${params.getUri}")
-      case TrackerResult.TrackerFailure(exception) =>
-        Logger[F].warn(exception)(
-          s"Telemetry heartbeat failed to reach ${params.getUri} after ${req.attempt} attempts"
-        )
-      case TrackerResult.RetriesExceeded(failure) =>
-        Logger[F].warn(s"Stopped trying to send telemetry heartbeat after failure: $failure")
-    }
+    httpClient
+      .run(request)
+      .use { response =>
+        if (response.status.isSuccess) Sync[F].unit
+        else {
+          Logger[F].info(show"Webhook ${request.uri} returned status ${response.status}")
+        }
+      }
+      .handleErrorWith { _ =>
+        // Don't log the exception, because http4s retry middleware already logs it
+        Logger[F].info(show"Webhook ${request.uri} resulted in exception without a response")
```

Comment:
Shouldn't we rather log them as `warn` ?

Reply:
This telemetry feature only impacts non-BDP pipelines.  And to be honest, I don't think that user group cares if a telemetry event fails.  They would probably prefer to have clean logs.  I'm almost tempted to go the other way and log it at debug level.